app/frontend/src/features/messages/Messages.test.tsx

```diff
@@ -15,17 +19,18 @@ afterEach(() => {
 });
 
 describe.each`
-  name                          | label            | count   | Component
-  ${"HostRequestsNotification"} | ${"Hosting"}     | ${"99"} | ${HostRequestsNotification}
-  ${"MessagesNotification"}     | ${"Group Chats"} | ${"99"} | ${MessagesNotification}
+  name                                  | label            | count   | Component
+  ${"HostRequestsSentNotification"}     | ${"Hosting"}     | ${"12"} | ${HostRequestsSentNotification}
+  ${"HostRequestsReceivedNotification"} | ${"Surfing"}     | ${"34"} | ${HostRequestsReceivedNotification}
+  ${"MessagesNotification"}             | ${"Group Chats"} | ${"56"} | ${MessagesNotification}
 `("$name", ({ label, count, Component }) => {
   it("shows the label with the number of unseen messages in a badge", async () => {
     pingMock.mockResolvedValue({
       user,
       pendingFriendRequestCount: 99,
-      unseenSentHostRequestCount: 99,
-      unseenReceivedHostRequestCount: 99,
-      unseenMessageCount: 99,
+      unseenSentHostRequestCount: 12,
+      unseenReceivedHostRequestCount: 34,
```

Comment:
These are failing atm

Reply:
Not sure how it wasn't failing when I push this up initially... but looks like they are catching the fact I got them the wrong way around which is why I changed the number in the first place aha. Will fix!